content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -56,22 +56,19 @@ array has six possible fields:
 * The `message` field is a human-readable message indicating details
   about the transition.
 
-A Pod has a PodStatus, which has an array of
-[PodConditions](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#podcondition-v1-core). Each element
-of the PodCondition array has a `type` field and a `status` field. The `type`
-field is a string with the following possible values:
-
-* `PodScheduled`: the Pod has been scheduled to a node;
-* `Ready`: the Pod is able to serve requests and should be added to the load
-  balancing pools of all matching Services;
-* `Initialized`: all [init containers](/docs/concepts/workloads/pods/init-containers)
-  have started successfully;
-* `Unschedulable`: the scheduler cannot schedule the Pod right now, for example
-  due to lacking of resources or other constraints;
-* `ContainersReady`: all containers in the Pod are ready.
-
-The `status` field is a string, with possible values "`True`", "`False`", and
-"`Unknown`".
+* The `reason` field is a unique, one-word, CamelCase reason for the condition's last transition
```

Comment:
How can it be unique and also be a reason? Surely it is one of a list of enumerated values?

Reply:
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/#podcondition-v1-core

Reply:
I see `Unique, one-word, CamelCase reason for the condition's last transition` but I don't get what it means. Can you show some examples? I don't see how a transition description can be unique. Surely there are only so many states to transition from or to?

Reply:
Ah, well, I didn't write the API documentation, just pointing to it

Reply:
Who would be the right person to ask for clarification?

Reply:
^ @kubernetes/sig-node-pr-reviews